app/main.py

```diff
@@ -1,2 +1,115 @@
+import copy
+from typing import Any, Hashable, Union, Iterable
+
+
+class Node:
+    def __init__(self, key: Any, value: Any) -> None:
+        self.key = key
+        self.value = value
+
+    def __repr__(self) -> str:
+        return f"{self.key} : {self.value}"
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.load_factor: float = 0.75
```

Comment:
load factor and other variables that cannot be changed better write like a class constant